tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/ProgrammeMembershipServiceImpl.java

```diff
@@ -132,10 +132,13 @@ public List<ProgrammeMembershipDTO> save(List<ProgrammeMembershipDTO> programmeM
       updatePersonWhenStatusIsStale(programmeMembership.getPerson().getId());
     });
 
-    //emit events
     List<ProgrammeMembershipDTO> resultDtos =
         programmeMembershipMapper
             .programmeMembershipsToProgrammeMembershipDTOs(programmeMemberships);
+
+    // update Person status
+    resultDtos.forEach(dto -> updatePersonWhenStatusIsStale(dto.getPerson().getId()));
```

Comment:
I'm not sure why we don't update person status when doing a bulk saving/updating.

Reply:
L#132 looks like we update the person status as soon as we save it.

Reply:
We can then refactor this block to nicely remove the `resultDtos` list definition as you did above.

Reply:
> L#132 looks like we update the person status as soon as we save it.

Ah, I must have got a fuzzy brain....You're right Pepe, will change it back.